src/components/organisms/AuthenticationModal/RegisterForm/RegisterForm.tsx

```diff
@@ -165,6 +169,39 @@ const RegisterForm: React.FunctionComponent<PropsType> = ({
             <span className="input-error">Password is required</span>
           )}
         </div>
+        <div className="input-group">
+          <input
+            name="date_of_birth"
+            className="input-block input-centered"
+            type="date"
+            ref={register}
+          />
+          {errors.date_of_birth && errors.date_of_birth.type === "required" && (
+            <span className="input-error">Date of birth is required</span>
+          )}
+        </div>
+        <div className="input-group">
+          <input
+            name="code"
+            className="input-block input-centered"
+            type="code"
+            placeholder="Ticket Code From Your Email"
+            ref={register({
+              required: true,
+            })}
+          />
+          {errors.code && (
+            <span className="input-error">
+              {errors.code.type === "required" ? (
+                <>
+                  Enter the ticket code from your email. The code is required.
+                </>
+              ) : (
+                errors.code.message
+              )}
+            </span>
+          )}
+        </div>
```

Comment:
This looks like a duplication of the code added in `src/components/organisms/AuthenticationModal/LoginForm/LoginForm.tsx`? If so, can we extract a reusable 'dumb' component or two?

Eg. something like:

```jsx
<TicketCodeField register={register} error={errors?.date_of_birth} />
<DateOfBirthField register={register} error={errors?.code} />
```